### DISCUSSION TITLE
How to select distinct records from datetime column with with only date not time?

### DISCUSSION THREAD

**Discussion by PawanGhewande** [simolus3/drift#1832] [Q&A] [answered]
I want to select distinct dates from column which contain the DateTime together. I want distinct dates only and ignore time.

**Comment by simolus3** [comment, marked as answer]
I think [`date`](https://drift.simonbinder.eu/api/drift/datetimeexpressions/date) might be what you're looking for? If you want to select distinct dates, you could use `selectOnly(yourTable, distinct: true)..addColumns([yourTable.dateColumns.date])`. You could also group by this expression if needed.
